<html>
<head>
<title>Fred P. Davis</title>
<STYLE TYPE="text/css">

<!--
BODY {
   font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif
/*   font: .8em Arial, Tahoma, Verdana;*/
}
-->
/* http://dustinfarris.com/2012/2/in-page-links-and-fixed-header-overlap/ */
span.anchor {
    display: block;
    height: 80px;
    margin-top: -80px;
    visibility: hidden;
}

/* http://www.webtutorialplus.com/sticky-header-on-scroll-jquery/ */

{
    margin: 0;
    padding: 0;
}
/*body
{
    font-size: 14px;
    color: #6a6272;
    font-family: Tahoma;
    background-image: url('bg-body.jpg');
}*/
.container
{
    width: 600px;
    margin: 0 auto;
    text-align: center;
    background-color: #B0CEFF;
}
.header
{
    padding: 10px 10% 10px 10%;*/
/*    padding: 10%;*/
    font-size: 20px;
    color: #666666;
    background-color: #FFFFFF;
}
.sticky-navigation
{
    padding: 0px;
    text-align: center;
/*    background: #666666;
    color: #FFFFFF;*/
    background: #FFFFFF;
    color: #000000;
    font-size: 18px;
    width: 100%;
    text-transform: uppercase;
}
.sticky-navigation.stuck
{
    padding: 0px;
    position: fixed;
    top: 0;
    width: 100%
/*    box-shadow: 0 2px 4px rgba(0, 0, 0, .2);*/
    box-shadow: 0 0px 0px rgba(0, 0, 0, .2);
}
.sticky-navigation ul
{
    border-style: solid;
    border-width: 1px 0px 1px 0px;
    border-color: #808080;
    padding: 2px;
/*    padding: 0px;*/
    list-style-type: none;    
}
.sticky-navigation ul li
{
    display: inline;
    padding: 15px;
    margin-left: auto;
    margin-right: auto;
}
.sticky-navigation a
{
    padding: 0px;
/*    color: #FFFFFF;*/
    color: #000000;
    text-decoration: none;
}
.sticky-navigation a:hover
{
/*    background: #999999;*/
    background: #B0C4DE;
}
.contents
{    
/*    color: #FFFFFF;*/
    color: #808080;
    font-size: 30px;   
    padding: 0px 10% 10px 10%;
}
.footer
{
    padding: 10px 10px 20px 10px;
    font-size: 16px;
/*    color: #FFFFFF;*/
/*    background-color: #999999;*/
    color: #000000;
    background-color: #B0C4DE;
}


</STYLE>
<script type="text/javascript" charset="utf8" src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.8.2.min.js"></script>

<script src="src/waypoints.min.js"></script>
<script src="src/waypoints-sticky.js"></script>
<script type="text/javascript">
//$(window).scroll(function () { 
// $('#test').css('background-color','#BC46'+($(this).scrollTop()/10))
//  });
        $(document).ready(function () {
            $('.sticky-navigation').waypoint('sticky');
        });
</script>

</head>
<body>

<div id="container">

<div class="header">
<table align="center" width=100%>
<tr><td align="left"><h2>Fred P. Davis</h2></td>
<td align="right" valign="top">Staff Scientist with <a href="http://irp.nih.gov/pi/john-oshea">John O'Shea</a>, <a href="http://www.nih.gov">NIH</a> <a href="http://www.niams.nih.gov">NIAMS</a></td></tr>
</table>
<p>
<table bgcolor="lightsteelblue" align="center">
<tr><td> <p align="justify">Every cell in our body has the same genome sequence, yet different kinds of cells express different parts of the genome. How does each cell know what genes to express?<br><br> <b>How is cell identity encoded in the genome?</b> I address this question in the nervous and immune systems &mdash; both of which have striking cell diversity &mdash; using genomic measurements and computation.</p></td></tr></table>
</div>

<div class="sticky-navigation">
<ul>
<li><a href="#projects">Projects</a></li>
<li><a href="#papers">Papers</a></li>
<li><a href="#software">Software</a></li>
<li><a href="#cv">CV</a></li>
<li><a href="#contact">Contact</a></li>
</ul>
</div>

<div class="contents">

<!--<table width=90% align="center">-->
<tr><td align="left">

<span class="anchor" id="projects"></span>
<h3>Projects</h3>

<!--<table bordercolor=black frame="above" cellpadding="5">-->

<table cellpadding="5">
<tr valign="top">
<td><p align="justify"><b>Cell type-specific genomics of neurons.</b> New technologies are enabling genomic measurements from individual cell types. In collaboration with <a href="http://janelia.org/people/lee-henry">Lee Henry</a> (<a href="http://janelia.org">HHMI Janelia</a>), we analyze expression and chromatin profiles from different neuronal subtypes to characterize the gene regulatory networks that underly cell identity.</p><br>

<p align="justify">&rarr; Transcription factors that regulate the identity of Kenyon cells in the <i>Drosophila</i> brain are expressed in these cells and repressed in octopaminergic neurons by the Polycomb-associated  H3K27me3 modification.</p>
<p style="font-size:small"><a href="#Mo_etal16">Mo <i>et al</i>. <i>eLife.</i> 2016</a><br>
<a href="#Mo_etal15">Mo, Mukamel, Davis, Luo, <i>et al</i>. <i>Neuron.</i> 2015</a><br>
<a href="#Henry_etal12">Henry, Davis, Picard, Eddy. <i>Nucleic Acids Res.</i> 2012</a></p>

</td>
<td width=40%><img width="300" src="figs/f3_intact.png"></td>
</tr>
</table>

<p>
<table bordercolor=black frame="above" cellpadding="5">
<tr valign="top">
<td width=40%><img width="350" src="figs/f1_transdiff.png">
<!--<p align="justify">&uarr; <i>SOX2</i>, <i>FOXG1</i>, and <i>POU3F2</i> are TFs that can convert fibroblasts (where they are more strongly Polycomb repressed) to neural stem cells (where they are more highly expressed). In contrast, <i>TWIST1</i> is a TF implicated in epithelial to mesenchymal transition, a transdifferentiation-like process involved in cancer formation.</p> 
<p style="font-size:small"><a href="#Davis_Eddy13">Davis and Eddy. <i>PLoS One</i>, in press</i></a></p>-->
</td>
<td><p align="justify"> <b>Identifying transcription factors that can convert cell identity.</b> Harold Weintraub and colleagues <a href="http://www.ncbi.nlm.nih.gov/pubmed/3690668">found nearly thirty years ago</a> that artifically expressing a single transcription factor, MyoD, in fibroblast cells can convert ("transdifferentiate") them to myoblasts. Since then, combinations of transcription factors have been discovered that can convert numerous other cell types, typically by empirically testing factors with a known developmental role in the goal cell type. We recently found that standard genomic methods (RNA-seq and ChIP-seq) can help identify conversion factors, as most are more strongly Polycomb repressed in the source cell and more highly expressed in the target cell. These results suggest that barriers between adult cell types, as depicted in <a href="http://www.ncbi.nlm.nih.gov/pubmed/12415319">Waddington's "epigenetic landscape"</a>, consist at least in part of differentially Polycomb-repressed transcription factors.</p>

</td></tr>

<tr valign="top">
<td colspan=2>
<p align="justify">&uarr; <i>SOX2</i>, <i>FOXG1</i>, and <i>POU3F2</i> are TFs that can convert fibroblasts (where they are more strongly Polycomb repressed) to neural stem cells (where they are more highly expressed). In contrast, <i>TWIST1</i> is a TF implicated in epithelial to mesenchymal transition, a transdifferentiation-like process involved in cancer formation.</p> 
<p style="font-size:small"><a href="#Davis_Eddy13">Davis and Eddy. <i>PLoS One</i>, 2013</i></a></p>
</td>
</tr>

</table>


<p>
<table bordercolor=black frame="above" cellpadding="5">
<tr valign="top">
<td><p align="justify"><b>Characterizing gene expression in novel brain nuclei.</b> Gene expression in the mouse brain has been systematically characterized by <i>in situ</i> hybridization (ISH), and registered in the three-dimensional <a href="http://mouse.brain-map.org">Allen Brain Atlas</a>. I developed a computer program, <a href="http://fredpdavis.com/allenminer">ALLENMINER</a>, that enables flexible searching of this expression data. In collaboration with <a href="http://www.unifr.ch/med/de/research/mc">Marco Celio</a> (<a href="http://www.unifr.ch">University of Fribourg</a>), I use this program to characterize previously undescribed regions in the mouse brain, such as the parvalbumin-positive (PV1) nucleus of the lateral hypothalamus. This analysis aims to characterize the gene batteries that define cell identity.</p>
</td>
<td width=50%><img width="75" src="figs/f2_pv1_roi.png"><img width="300" src="figs/f2_pv1_ish.png">
<p align="justify">&uarr; Candidate genes that express in the PV1 nucleus (green box) of the hypothalamus (shown in blue wireframe) were identified by ALLENMINER and validated by ISH.</p>
<p style="font-size:small"><a href="#Davis_Eddy09">Davis and Eddy. <i>Bioinformatics</i> 2009.<a/><br><a href="#Girard_etal11">Girard, Meszar, Marti, Davis, Celio. <i>Eur J Neurosci</i> 2011.</a></p>

</td>
</tr>
</table>


<span class="anchor" id="papers"></span>
<h3>Papers</h3>

<table align="center">
<tr><td align="left">

<ol>

<span class="anchor" id="Pregger_etal16"></span>
<li value="25">
<a href="http://dx.doi.org/10.1016/j.devcel.2016.10.010">Evolved Repression Overcomes Enhancer Robustness.</a><br>
Preger-Ben Noon E, <b>Davis FP</b>, Stern DL. <i>Dev Cell</i> (2016). doi:10.1016/j.devcel.2016.10.010. (<a href="papers/Preger2016DevCell.pdf">reprint</a>).</li><br>

<span class="anchor" id="Shih_etal16"></span>
<li value="24">
<a href="http://dx.doi.org/10.1016/j.cell.2016.04.029">Developmental acquisition of regulomes underlies innate lymphoid cell functionality.</a><br>
Shih HY*, Sciume G*, Mikami Y*, Guo L, Sun HW, Brooks SR, Urban Jr JF, <b>Davis FP</b>, Kanno Y, O'Shea JJ. <i>Cell</i> (2016). doi:10.1016/j.cell.2016.04.029. (* co-first authors). (<a href="papers/Shih2016Cell.pdf">reprint</a>).</li><br>

<span class="anchor" id="Mo_etal16"></span>
<li value="23">
<a href="http://dx.doi.org/10.7554/eLife.11613">Epigenomic landscapes of retinal rods and cones.</a><br>
Mo A, Luo C, <b>Davis FP</b>, Mukamel EA, Henry GL, Nery JR, Urich MA, Picard S, Lister R, Eddy SR, Beer MA, Ecker JR, Nathans J. <i>eLife</i> (2016). 5:e11613. (<a href="papers/Mo2016eLife.pdf">reprint</a>).<br>
Raw data: <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72550">GEO GSE72550</a></li></br>

<span class="anchor" id="Davis_etal15"></span>
<li value="22">
<a href="http://dx.doi.org/10.1016/j.cell.2015.11.033">A metabolic switch for Th17 pathogenicity.</a><br>
<b>Davis FP</b>, Kanno Y, O'Shea JJ. <i>Cell</i> (2015). 163:1308-1310 (commentary). (<a href="papers/Davis2015Cell.pdf">reprint</a>).</li><br>

<span class="anchor" id="Mo_etal15"></span>
<li value="21">
<a href="http://dx.doi.org/10.1016/j.neuron.2015.05.018">Epigenomic signatures of neuronal diversity in the mammalian brain.</a><br>
Mo A*, Mukamel EA*, <b>Davis FP*</b>, Luo C*, Henry GL, Picard S, Urich MA, Nery JR, Sejnowski TJ, Lister R, Eddy SR, Ecker JR, Nathans J. <i>Neuron</i>. (2015) 86(6): 1369-1384. (* co-first authors). (<a href="papers/Mo2015Neuron.pdf">reprint</a>)<br>
Selected as Issue Highlight, <a href="http://www.cell.com/neuron/abstract/S0896-6273(15)00515-2">previewed by Steve Henikoff</a>.<br>
Raw data: <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63137">GEO GSE63137</a></li></br>

<li value="20">
<a href="http://www.karger.com/Article/FullText/360152">NSF Workshop Report: Discovering General Principles of Nervous System Organization by Comparing Brain Maps Across Species.</a><br>
Striedter GF, Belgard TG, Chen C, <b>Davis FP</b>, Finlay BL, Gunturkun O, Hale ME, Harris J, Hecht EE, Hof PR, <i>et al</i>.
<i>Brain, Behavior and Evolution</i> (2014). 83: 1-8. doi:10.1159/000360152 (<a href="papers/Striedter2014BBE.pdf">reprint</a>).</li><br>

<span class="anchor" id="Davis_Eddy13"></span>
<li value="19">
<a href="http://dx.doi.org/10.1371/journal.pone.0063407">Transcription factors that convert adult cell identity are differentially Polycomb repressed.</a><br>
<b>Davis FP</b> and Eddy SR. <i>PLoS One</i> (2013). 8(5): e63407. doi:10.1371/journal.pone.0063407 (<a href="papers/Davis2013PLoSOne.pdf">reprint</a>).</li><br>

<span class="anchor" id="Henry_etal12"></span>
<li value="18">
<a href="http://dx.doi.org/10.1093/nar/gks671">Cell-type specific genomics of Drosophila neurons.</a><br>
Henry GL, <b>Davis FP</b>, Picard S, Eddy SR. <i>Nucleic Acids Res</i> (2012). 40 (19): 9691-9704. (<a href="papers/Henry2012NAR.pdf">reprint</a>).<br>Designated feature article.<br>
Raw data: <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37027">GEO GSE37027</a></li><br>

<li value="17">
<a href="http://dx.doi.org/10.1016/j.str.2011.11.006">Phosphorylation at the interface.</a><br>
<b>Davis FP</b>. <i>Structure </i> (2011). 19(12):1726-1727 (commentary). (<a href="papers/Davis2011Structure.pdf">reprint</a>).</li><br>

<span class="anchor" id="Girard_etal11"></span>
<li value="16">
<a href="http://dx.doi.org/10.1111/j.1460-9568.2011.07918.x">Gene expression analysis in the parvalbumin-immunoreactive PV1 nucleus of the mouse lateral hypothalamus.</a><br>
Girard F, Meszar Z, Marti C, <b>Davis FP</b>, Celio M. <i>Eur. J. Neurosci</i> (2011). 34(12): 1934-1943. (<a href="papers/Girard2011EurJNeurosci.pdf">reprint</a>).</li><br>

<li value="15">
<a href="http://dx.doi.org/10.1039/C0MB00200C">Proteome-wide prediction of overlapping small molecule and protein binding sites using structure.</a><br>
<b>Davis FP</b>. <i>Molecular Biosystems</i> (2011). 7(2): 545-547. (<a href="papers/Davis2011MolBiosyst.pdf">reprint</a>).<br>
Selected for the <a href="http://scitation.aip.org/getabs/servlet/GetabsServlet?prog=normal&id=VIRT02000021000003000041000001&idtype=cvips&gifs=yes&ref=no"><i>Virtual Journal of Biological Physics Research</i></a>.<br>
Software: <a href="http://fredpdavis.com/homolobind">HOMOLOBIND</a></li><br>

<li value="14">
<a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1000668">The overlap of small molecule and protein binding sites within families of protein structures.</a><br>
<b>Davis FP</b> and Sali A. <i>PLoS Computational Biology</i> (2010) 6(2): e1000668. (<a href ="papers/Davis2010PLoSCompBiol.pdf">reprint</a>).<br> Designated feature research.<br>
Database: <a href="http://fredpdavis.com/pibase.ligands">PIBASE.ligands</a></li><br>

<span class="anchor" id="Davis_Eddy09"></span>
<li value="13">
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btp288">A tool for identification of genes expressed in patterns of interest using the Allen Brain Atlas.</a><br>
<b>Davis FP</b> and Eddy SR. <i>Bioinformatics</i> (2009) 25(13):1647-54. (<a href="papers/Davis2009Bioinf.pdf">reprint</a>).<br>
Software: <a href="http://fredpdavis.com/allenminer">ALLENMINER</a></li><br>

<li value="12">
<a href="http://dx.doi.org/10.1093/nar/gkn791">MODBASE, a database of annotated comparative protein structure models and associated resources</a><br>
Pieper U, Eswar N, Webb BM, Eramian D, Kelly L, Barkan DT, Carter H, Mankoo P, Karchin R, Marti-Renom MA, <b>Davis FP</b>, Sali A. <i>Nucleic Acids Res</i> (2008). (<a href="papers/Pieper2008NAR.pdf">reprint</a>).<br>
Database: <a href="http://salilab.org/modbase">MODBASE</a></li><br>

<li value="11">
<a href="http://www.proteinscience.org/cgi/doi/10.1110/ps.073228407">Host-pathogen protein interactions predicted by comparative modeling.</a><br>
<b>Davis FP</b>, Barkan DT, Eswar N, McKerrow JH, Sali A. <i>Protein Science</i> (2007) 16: 2585-2596. (<a href="papers/Davis2007ProteinSci.pdf">reprint</a>) (<a href="papers/Davis2007ProteinSci_suppl.pdf">supplementary text</a>).<br>
Software: <a href="http://fredpdavis.com/modtie">MODTIE</a>.<br>
Data: <a href="http://dx.doi.org/10.5281/zenodo.29608"><img src="https://zenodo.org/badge/doi/10.5281/zenodo.29608.svg"/></a></li><br>

<li value="10">
<a href ="http://dx.doi.org/10.1093/nar/gkm236">DBAli tools: mining the protein structure space.</a><br>
Marti-Renom MA, Pieper U, Madhusudhan MS, Rossi A, Eswar N, <b>Davis FP</b>, Al-Shahrour F, Dopazo J, and Sali A. <i>Nucleic Acids Research</i> (2007) 35: W393-7. (<a href="papers/MartiRenom2007NAR.pdf">reprint</a>).</li><br>

<li value="9">
<a href ="http://www.biomedcentral.com/1471-2105/8/S4/S4">The AnnoLite and AnnoLyze Programs for Comparative Annotation of Protein Structures.</a><br>
Marti-Renom MA, Rossi A, Al-Shahrour F, <b>Davis FP</b>, Pieper U, Dopazo J, and Sali A. <i>BMC Bioinformatics</i> (2007) 8:S4. (<a href="papers/MartiRenom2007BMCBioinf.pdf">reprint</a>)</li><br>

<li value="8">
<a href ="http://dx.doi.org/10.1371/journal.pcbi.0020153">Modeling protein interactions by analogy: Application to PSD-95.</a><br>
Korkin D, <b>Davis FP</b>, Alber A, Luong T, Shen MY, Lucic V, Kennedy MB, Sali A. <i>PLoS Computational Biology</i> (2006) 2: e153. (<a href="papers/Korkin2006PLoSCompBiol.pdf">reprint</a>)</li><br>

<li value="7">
<a href ="http://nar.oxfordjournals.org/cgi/content/full/34/10/2943">Protein complex compositions predicted by structural similarity.</a><br>
<b>Davis FP</b>, Braberg H, Shen MY, Pieper U, Madhusudhan MS, Sali A. <i>Nucleic Acids Research</i> (2006) 34:2943-2952. (<a href="papers/Davis2006NAR.pdf">reprint</a>).<br>
Software: <a href="http://fredpdavis.com/modtie">MODTIE</a></li><br>


<li value="6">
<a href ="http://dx.doi.org/10.1093/nar/gkj059">MODBASE: a database of annotated comparative protein structure models and associated resources.</a><br>
Pieper U, Eswar N, <b>Davis FP</b>, Braberg H, Madhusudhan MS, Rossi A, Marti-Renom MA, Karchin R, Webb BM, Eramian D, Shen MY, Kelly L, Melo F, Sali A. <i>Nucleic Acids Research</i> (2006) 34: D291-D295. (<a href="papes/Pieper2006NAR.pdf">reprint</a>).<br>
Database: <a href="http://salilab.org/modbase">MODBASE</a></li><br>


<li value="5">
<a href ="http://dx.doi.org/10.1110/ps.051571905">Localization of protein-binding sites within families of proteins.</a><br>
Korkin D, <b>Davis FP</b>, Sali A.  <i>Protein Science</i> (2005) 14: 2350-2360. (<a href="papers/Korkin2005ProteinSci.pdf">reprint</a>) </li><br>

<li value="4">
<a href ="http://dx.doi.org/10.1016/j.cplett.2005.02.029"> The Optimal Size of a Globular Protein Domain: A Simple Sphere-Packing Model.</a><br>
Shen MY, <b>Davis FP</b>, Sali A. <i>Chemical Physics Letters</i> (2005) 405: 224-228. (<a href="papers/Shen2005ChemPhysLett.pdf">reprint</a>) </li><br>

<li value="3">
<a href="http://dx.doi.org/10.1093/bioinformatics/bti277">PIBASE: a comprehensive database of structurally defined protein interfaces.</a><br>
<b>Davis FP</b> and Sali A. <i>Bioinformatics</i> (2005) 21(9): 1901-1907. (<a href="papers/Davis2005Bioinf.pdf">reprint</a>).<br>
Database: <a href="http://fredpdavis.com/pibase">PIBASE</a>.</li><br>

<li value="2">
<a href="http://dx.doi.org/10.1016/j.sbi.2004.04.006">A structural perspective on protein-protein interactions.</a><br>
Russell RB, Alber F, Aloy P, <b>Davis FP</b>, Korkin D, Pichaud M, Topf M, Sali A. <i>Current Opinions in Structural Biology</i> (2004) 14(3): 313-24. Review. (<a href="papers/Russell2004COSB.pdf">reprint</a>) </li><br>

<li value="1">
<a href="http://dx.doi.org/10.1093/nar/gkh095">MODBASE, a database of annotated comparative protein structure models, and associated resources.</a><br>
Pieper U, Eswar N, Braberg H, Madhusudhan MS, <b>Davis FP</b>, Stuart AC, Mirkovic N, Rossi A, Marti-Renom MA, Fiser A, Webb B, Greenblatt D, Huang CC, Ferrin TE, Sali A. <i>Nucleic Acids Res</i> (2004) 32 Database issue: D217-22. (<a href="papers/Pieper2004NAR.pdf">reprint</a>).<br>
Database: <a href="http://salilab.org/modbase">MODBASE</a></li><br>
</ol>

</td>
</tr>
</table>

<h3 id="software">Software</h3>

<table align="left"><tr><td align="left">
I've written several programs, mostly for protein structure and gene expression analysis, and all are <a href="http://github.com/fredpdavis?tab=repositories">available at github</a>.
<p>
</td></tr></table>

<table align="center" width=100%>

<tr>
<td align="right"><a href="http://fredpdavis.com/mdlabbook"><img width=60 style="border-width:0px" src="https://avatars2.githubusercontent.com/u/13946400?v=3&s=460"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/mdlabbook">mdllabbook</a> helps maintain a markdown-format lab notebook.</br>
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://fredpdavis.com/allenminer"><img width=60 style="border-width:0px" src="images/allenminer_web_logo.png"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/allenminer">ALLENMINER</a> - A tool to identify genes expressed in patterns of interest using the Allen Brain Atlas.<br>
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://fredpdavis.com/pibase"><img width=60 style="border-width:0px" src="images/pibase_blue_web.png"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/pibase">PIBASE</a> - A comprehensive relational database of structurally defined protein interfaces.<br>
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://fredpdavis.com/pibase.ligands"><img style="border-width:0px" width=60 src="images/pilig_web_logo.png"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/pibase.ligands">PIBASE.ligands</a> - Overlapping small molecule and protein binding sites.<br>
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://fredpdavis.com/modtie"><img style="border-width:0px" width=60 src="images/modtie_web_logo.png"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/modtie">MODTIE</a> - Software to predict protein complex compositions using structural similarity.<br>
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://dx.doi.org/10.5281/zenodo.29608"><img style="border-width:0px" width=60 src="images/hp_logo_web.png"></a></td>
<td><DIV class=bodytext>
<a href="http://dx.doi.org/10.5281/zenodo.29608">Predicted host-pathogen interactions</a> - Host-pathogen protein interactions predicted by comparative modeling.
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://fredpdavis.com/homolobind"><img style="border-width:0px" width=60 src="images/homolobind_logo.png"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/homolobind">HOMOLOBIND</a> - proteome-wide prediction of protein and ligand binding sites using structure.
</DIV></td>
</tr>

</table>

<h3 id="cv">CV: <a href="files/fred_p_davis_cv.pdf">download</a></h3>


<!--</table>-->
</div>
<div class="footer">
<h3 id="contact">Contact:</h3>
post: Lymphocyte Cell Biology Section, NIAMS, National Institutes of Health, Bethesda, MD 20892, USA <br>
email: fredpdavis at gmail dot com <br>
phone: (+1) 619-929-5892

</div>
</div>

</body>
</html>
