<html>
<head>
<title>Fred P. Davis</title>
<STYLE TYPE="text/css">

<!--
BODY {
   font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif
/*   font: .8em Arial, Tahoma, Verdana;*/
}
-->
/* http://dustinfarris.com/2012/2/in-page-links-and-fixed-header-overlap/ */
span.anchor {
    display: block;
    height: 80px;
    margin-top: -80px;
    visibility: hidden;
}

/* http://www.webtutorialplus.com/sticky-header-on-scroll-jquery/ */

{
    margin: 0;
    padding: 0;
}
/*body
{
    font-size: 14px;
    color: #6a6272;
    font-family: Tahoma;
    background-image: url('bg-body.jpg');
}*/
.container
{
    width: 600px;
    margin: 0 auto;
    text-align: center;
    background-color: #B0CEFF;
}
.header
{
    padding: 10px 10% 10px 10%;*/
/*    padding: 10%;*/
    font-size: 20px;
    color: #666666;
    background-color: #FFFFFF;
}
.sticky-navigation
{
    padding: 0px;
    text-align: center;
/*    background: #666666;
    color: #FFFFFF;*/
    background: #FFFFFF;
    color: #000000;
    font-size: 18px;
    width: 100%;
    text-transform: uppercase;
}
.sticky-navigation.stuck
{
    padding: 0px;
    position: fixed;
    top: 0;
    width: 100%
/*    box-shadow: 0 2px 4px rgba(0, 0, 0, .2);*/
    box-shadow: 0 0px 0px rgba(0, 0, 0, .2);
}
.sticky-navigation ul
{
    border-style: solid;
    border-width: 1px 0px 1px 0px;
    border-color: #808080;
    padding: 2px;
/*    padding: 0px;*/
    list-style-type: none;    
}
.sticky-navigation ul li
{
    display: inline;
    padding: 15px;
    margin-left: auto;
    margin-right: auto;
}
.sticky-navigation a
{
    padding: 0px;
/*    color: #FFFFFF;*/
    color: #000000;
    text-decoration: none;
}
.sticky-navigation a:hover
{
/*    background: #999999;*/
    background: #B0C4DE;
}
.contents
{    
/*    color: #FFFFFF;*/
    color: #808080;
    font-size: 30px;   
    padding: 0px 10% 10px 10%;
}
.footer
{
    padding: 10px 10px 20px 10px;
    font-size: 16px;
/*    color: #FFFFFF;*/
/*    background-color: #999999;*/
    color: #000000;
    background-color: #B0C4DE;
}


</STYLE>
<script type="text/javascript" charset="utf8" src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.8.2.min.js"></script>

<script src="src/waypoints.min.js"></script>
<script src="src/waypoints-sticky.js"></script>
<script type="text/javascript">
//$(window).scroll(function () { 
// $('#test').css('background-color','#BC46'+($(this).scrollTop()/10))
//  });
        $(document).ready(function () {
            $('.sticky-navigation').waypoint('sticky');
        });
</script>

</head>
<body>

<div id="container">

<div class="header">
<table align="center" width=100%>
<tr><td align="left"><h2>Fred P. Davis</h2></td>
<td align="right" valign="top">This personal website does not reflect the views of my employer.</td></tr>
</table>
<p>
<table bgcolor="lightsteelblue" align="center">
<tr><td> <p align="justify">Every cell in our body has the same genome, yet different kinds of cells express different parts of it. How does this work?<br><br> <b>How is cell identity encoded in the genome?</b> I explore this question in the nervous and immune systems &mdash; both of which have striking cell diversity &mdash; using genomic measurements and computation.</p></td></tr></table>
</div>

<div class="sticky-navigation">
<ul>
<li><a href="#bio">Bio</a></li>
<li><a href="#projects">Projects</a></li>
<li><a href="#papers">Papers</a></li>
<li><a href="#software">Code</a></li>
<li><a href="#contact">Contact</a></li>
</ul>
</div>

<div class="contents">

<!--<table width=90% align="center">-->
<tr><td align="left">

<span class="anchor" id="bio"></span>
<h3 id="bio">Bio</h3>


<table cellpadding="5"> <tr valign="top"> <td align="left">
         As a quantitative biologist, I enjoy solving biomedical puzzles with computation, statistics, and visualization.<br><br>

         I work at the <a href="http://www.nih.gov">NIH</a> as a Staff Scientist in <a href="http://irp.nih.gov/pi/john-oshea">John O'Shea's</a> lab for lymphocyte cell biology. I trained in computational genomics with <a href="http://eddylab.org">Sean Eddy</a> at <a href="http://janelia.org">HHMI Janelia</a> (Postdoc, Staff 2007-2015) and computational structural biology with <a href="http://salilab.org">Andrej Sali</a> at <a href="http://www.ucsf.edu">UCSF</a> (Ph.D., Biophysics. 2003-2007). Before that I studied pharmacy (Pharm.D., <a href="http://www.pharmacy.purdue.edu">Purdue</a> 2001).

   <a href="files/fred_p_davis_cv.pdf">Download CV</a></h3>
</td></tr>
</table>


<span class="anchor" id="projects"></span>
<h3>Recent Projects</h3>

<!--<table bordercolor=black frame="above" cellpadding="5">-->

<table cellpadding="5">
<tr valign="top">
   <td><p align="justify"><b>Using gene expression to explore neural circuit function.</b> Different neurons express different sets of genes, but can this information help us understand how neural circuits work? To explore this question, we turned to the <i>Drosophila</i> visual system, a well-established model for studying the development, genetics, and structure of neural circuits. Together with <a href="http://janelia.org/people/aljoscha-nern">Aljoscha Nern</a> (<a href="http://janelia.org">Janelia</a>) and <a href="https://www.cshl.edu/research/faculty-staff/">Lee Henry</a> (Janelia, now CSHL) we combined genetic tools to access individual cell types, a method to profile their nuclear transcriptomes, and a probabilistic model to interpret these measurements. We found that this information indeed helps illuminate circuit function, ranging from fundamental properties such as the neurotransmitters used at each synapse to higher-level circuit feedback mechanisms.</p></br>
<p align="justify">&rarr; Gene expression levels measured by TAPIN-seq allow the inference of neurotransmitters produced by visual system neurons (row=cell type, column=gene).</p>
<a href="#Davis_etal18">Davis & Nern <i>et al.</i> 2018.</a></p>
</td>
<td width=50% align="center"><img width="300" src="figs/davisnern18_fig5a.png"></td>
</tr>

</table>

<p>
<table bordercolor=black frame="above" cellpadding="5">
<tr valign="top">
<td width=40%><img width="450" src="figs/iwata_fig2c.png">
</td>
<td><p align="justify"> <b>A lineage-determining transcription factor also regulates signal response</b>. T-bet is the master transcription factor that specifies the T helper 1 lineage of T lymphocytes. Using a suite of genomic approaches, we found an additional and unexpected role for T-bet in repressing the Type I Interferon expression program.<br><br>&larr; In the absence of T-bet (red), T cells incorrectly respond to interferon-&gamma; (orange) as though it was interferon-&beta; (blue).</br>

<p><a href="#Iwata_etal17">Iwata & Mikami, ..., O'Shea J+, Davis FP+, Kanno Y+. <i>Immunity</i>, 2017</i></a></p>
</td>
</tr>

</table>


<p>
<table bordercolor=black frame="above" cellpadding="5">
<tr valign="top">
   <td><p align="justify"><b>Mapping the epigenetic and transcriptional landscape of neurons.</b> Together with <a href="https://www.cshl.edu/research/faculty-staff/">Lee Henry</a> (Janelia, now CSHL), we developed a genetic approach to measure genomic properties in individual neuronal cell types. This technology reveals distinct gene expression, histone modification, and chromatin accessibility landscapes. We first demonstrated this technology in the <i>Drosophila</i> brain, using Polycomb-associated histone marks and gene expression to predict TFs critical for cell identity. In collaboration with Alisa Mo (<a href="http://nathanslab.mbg.jhmi.edu">Jeremy Nathans lab, Hopkins</a>), we demonstrated the ability of this approach to profile neuronal cell types in the mouse cortex.
   <p align="justify">&rarr; Cell type-specific genomics identifies footprints of transcription factor (TF) binding and predicts TF regulatory networks active in specific cell types, such as VIP inhibitory neurons.</p>
<a href="#Mo_etal15">Mo*, Mukamel*, Davis*, Luo*, <i>et al</i>. <i>Neuron.</i> 2015</a><br>
<a href="#Henry_etal12">Henry, Davis, Picard, Eddy. <i>NAR.</i> 2012</a><br>
</td>
<td width=55% align="left"><img width="250" src="figs/mo15_fig5a.png"><img width="200" src="figs/mo15_fig5d.png"></td>
</tr>
</table>


<span class="anchor" id="papers"></span>
<h3>Papers</h3>


<table align="center">
<tr><td align="left">

      <a href="https://scholar.google.com/citations?user=7IwdznQAAAAJ&hl=en">Google Scholar</a></br>
      <br>
* co-first author, + corresponding authors<br>
<ol>
<span class="anchor" id="Nagashima_etal19"></span>
<li value="34">
<a href="http://dx.doi.org/10.1016/j.immuni.2019.06.009">Neuropeptide CGRP Limits Group 2 Innate Lymphoid Cell Responses and Constrains Type 2 Inflammation.</a><br>
Nagashima H, Mahlak&otilde;iv T, Shih HY, <b>Davis FP</b>, Meylan F, Huang Y, Harrison OJ, Yao C, Mikami Y, Urban JF Jr, Caron KM, Belkaid Y, Kanno Y, Artis D, O'Shea JJ. <i>Immunity</i>. (2019) online ahead of print. (<a href="papers/Nagashima2019Immunity.pdf">reprint</a>).<br>
Data: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131996">NCBI GEO GSE131996</a></li><br>

<span class="anchor" id="Schwartz_etal19"></span>
<li value="33">
<a href="http://dx.doi.org/10.1016/j.immuni.2018.12.014">Retinoic Acid Receptor Alpha Represses a Th9 Transcriptional and Epigenomic Program to Reduce Allergic Pathology.</a><br>
Schwartz DM, Farley TK, Richoz N, Yao C, Shih HY, Petermann F, Zhang Y, Sun HW, Hayes E, Mikami Y, Jiang K, <b>Davis FP</b>, Kanno Y, Milner JD, Siegel R, Laurence A, Meylan F, O'Shea JJ. <i>Immunity</i>. (2019) 50 (1): 106-120. (<a href="papers/Schwartz2019Immunity.pdf">reprint</a>).<br>
Data: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123501">NCBI GEO GSE123501</a></li><br>

<span class="anchor" id="Shih_etal18"></span>
<li value="32">
<a href="https://dx.doi.org/10.1534/g3.118.200726">Nuclear transcriptomes of the seven neuronal cell types that constitute the Drosophila mushroom bodies.</a><br>
Shih MF*, <b>Davis FP*</b>, Henry GL+, Dubnau J+. <i>G3</i>. (2019) 9 (1): 81-94. (<a href="papers/Shih2018G3_early.pdf">reprint</a>).<br>
Preprint: <a href="https://doi.org/10.1101/412569">bioRxiv 412569</a>. (<a href="papers/Shih2018bioRxiv.pdf">preprint</a>).<br>
Code: <a href="http://github.com/fredpdavis/mushroombody">github</a>.<br>
Data: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119629">NCBI GEO GSE119629</a></li><br>

<span class="anchor" id="Davis_etal18"></span>
<li value="31">
<a href="https://doi.org/10.1101/385476">A genetic, genomic, and computational resource for exploring neural circuit function.</a><br>
<b>Davis FP*</b>, Nern A*, Picard S, Reiser MB, Rubin GM, Eddy SR, Henry GL. <i>under review</i>.<br>
Preprint: <a href="https://doi.org/10.1101/385476">bioRxiv 385476</a>. (<a href="papers/Davis2018bioRxiv.pdf">preprint</a>).<br>
Code: <a href="http://github.com/fredpdavis/opticlobe">github</a>. Data: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116969">NCBI GEO GSE116969</a></li><br>

<span class="anchor" id="Bentzur_etal18"></span>
<li value="30">
<a href="https://doi.org/10.1371/journal.pgen.1007328">Odorant binding protein 69a connects social interaction to modulation of social responsiveness in Drosophila.</a><br>
Bentzur A, Shmueli A, Omesi L, Ryvkin J, Knapp JM, Parnas M, <b>Davis FP</b>, Shohat-Ophir G. <i>PLoS Genetics</i> (2018) 14(4): e1007328. (<a href="papers/Bentzur2018PLoSGenetics.pdf">reprint</a>).</li><br>

<span class="anchor" id="Xie_etal17"></span>
<li value="29">
<a href="http://dx.doi.org/10.1101/gad.303321.117">A dynamic interplay of enhancer elements regulates Klf4 expression in naïve pluripotency.</a><br>
Xie L, Torigoe SE, Xiao J, Mai DH, Li L, <b>Davis FP</b>, Dong P, Marie-Nelly H, Grimm J, Lavis L, Darzacq X, Cattoglio C, Liu Z, Tjian R. <i>Genes Dev</i> (2017) 31(17):1795-1808. (<a href="papers/Xie2017GenesDev.pdf">reprint</a>).</li><br>

<span class="anchor" id="Villarino_etal17"></span>
<li value="28">
<a href="http://dx.doi.org/10.1084/jem.20150907">Subset- and tissue-defined STAT5 thresholds control homeostasis and function of innate lymphoid cells.</a><br>
Villarino AV*, Sciume G*, <b>Davis FP</b>, Iwata S, Zitti B, Robinson GW, Hennighausen L, Kanno Y, O'Shea JJ. <i>J Exp Med</i> (2017) 214(10:2999-3014. (<a href="papers/Villarino2017JEM.pdf">reprint</a>).</li><br>

<span class="anchor" id="Iwata_etal17"></span>
<li value="27">
<a href="http://dx.doi.org/10.1016/j.immuni.2017.05.005">The transcription factor T-bet limits amplification of type I IFN transcriptome and circuitry in T helper 1 cells.</a><br>
Iwata S*, Mikami Y*, &hellip;, O’Shea JJ+, <b>Davis FP+</b>, Kanno Y+. <i>Immunity</i> (2017)  46(6): p983-991.e4. (<a href="papers/Iwata2017Immunity.pdf">reprint</a>).<br>
<a href="http://dx.doi.org/10.1016/j.immuni.2017.05.010">Preview by Lazarevic, Szabo, and Glimcher: T-bet Runs INTERFERence</a>.<br>
Data: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96724">GEO GSE96724</a></li></br>

<span class="anchor" id="Afzali_etal17"></span>
<li value="26">
<a href="http://dx.doi.org/10.1038/ni.3753">BACH2 immunodeficiency illustrates an association between super-enhancers and haploinsufficiency.</a><br>
Afzali B*, Gronholm J*, Vandrocova J*, &hellip; <b>Davis FP</b> &hellip;, Cooper N+, Laurence ADJ+. <i>Nature Immunology</i> (2017) 18: 813-823. (<a href="papers/Afzali2017NI.pdf">reprint</a>).</li><br>

<span class="anchor" id="Pregger_etal16"></span>
<li value="25">
<a href="http://dx.doi.org/10.1016/j.devcel.2016.10.010">Evolved Repression Overcomes Enhancer Robustness.</a><br>
Preger-Ben Noon E, <b>Davis FP</b>, Stern DL. <i>Dev Cell</i> (2016) 39(5): 572-584. (<a href="papers/Preger2016DevCell.pdf">reprint</a>).</li><br>

<span class="anchor" id="Shih_etal16"></span>
<li value="24">
<a href="http://dx.doi.org/10.1016/j.cell.2016.04.029">Developmental acquisition of regulomes underlies innate lymphoid cell functionality.</a><br>
Shih HY*, Sciume G*, Mikami Y*, Guo L, Sun HW, Brooks SR, Urban Jr JF, <b>Davis FP</b>, Kanno Y, O'Shea JJ. <i>Cell</i> (2016) 165 (5): 1120-1133. (<a href="papers/Shih2016Cell.pdf">reprint</a>).</li><br>

<span class="anchor" id="Mo_etal16"></span>
<li value="23">
<a href="http://dx.doi.org/10.7554/eLife.11613">Epigenomic landscapes of retinal rods and cones.</a><br>
Mo A, Luo C, <b>Davis FP</b>, Mukamel EA, Henry GL, Nery JR, Urich MA, Picard S, Lister R, Eddy SR, Beer MA, Ecker JR, Nathans J. <i>eLife</i> (2016). 5:e11613. (<a href="papers/Mo2016eLife.pdf">reprint</a>).<br>
Data: <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72550">GEO GSE72550</a></li></br>

<span class="anchor" id="Davis_etal15"></span>
<li value="22">
<a href="http://dx.doi.org/10.1016/j.cell.2015.11.033">A metabolic switch for Th17 pathogenicity.</a><br>
<b>Davis FP+</b>, Kanno Y, O'Shea JJ+. <i>Cell</i> (2015). 163:1308-1310 (commentary). (<a href="papers/Davis2015Cell.pdf">reprint</a>).</li><br>

<span class="anchor" id="Mo_etal15"></span>
<li value="21">
<a href="http://dx.doi.org/10.1016/j.neuron.2015.05.018">Epigenomic signatures of neuronal diversity in the mammalian brain.</a><br>
Mo A*, Mukamel EA*, <b>Davis FP*</b>, Luo C*, Henry GL, Picard S, Urich MA, Nery JR, Sejnowski TJ, Lister R, Eddy SR, Ecker JR, Nathans J. <i>Neuron</i>. (2015) 86(6): 1369-1384. (<a href="papers/Mo2015Neuron.pdf">reprint</a>)<br>
Selected as Issue Highlight, <a href="http://www.cell.com/neuron/abstract/S0896-6273(15)00515-2">previewed by Steve Henikoff</a>.<br>
Data: <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63137">GEO GSE63137</a></li></br>

<li value="20">
<a href="http://www.karger.com/Article/FullText/360152">NSF Workshop Report: Discovering General Principles of Nervous System Organization by Comparing Brain Maps Across Species.</a><br>
Striedter GF, Belgard TG, Chen C, <b>Davis FP</b>, Finlay BL, Gunturkun O, Hale ME, Harris J, Hecht EE, Hof PR, <i>et al</i>.
<i>Brain, Behavior and Evolution</i> (2014). 83: 1-8. doi:10.1159/000360152 (<a href="papers/Striedter2014BBE.pdf">reprint</a>).</li><br>

<span class="anchor" id="Davis_Eddy13"></span>
<li value="19">
<a href="http://dx.doi.org/10.1371/journal.pone.0063407">Transcription factors that convert adult cell identity are differentially Polycomb repressed.</a><br>
<b>Davis FP</b> and Eddy SR. <i>PLoS One</i> (2013). 8(5): e63407. doi:10.1371/journal.pone.0063407 (<a href="papers/Davis2013PLoSOne.pdf">reprint</a>).</li><br>

<span class="anchor" id="Henry_etal12"></span>
<li value="18">
<a href="http://dx.doi.org/10.1093/nar/gks671">Cell-type specific genomics of Drosophila neurons.</a><br>
Henry GL, <b>Davis FP</b>, Picard S, Eddy SR. <i>Nucleic Acids Res</i> (2012). 40 (19): 9691-9704. (<a href="papers/Henry2012NAR.pdf">reprint</a>).<br>Designated feature article.<br>
Data: <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37027">GEO GSE37027</a></li><br>

<li value="17">
<a href="http://dx.doi.org/10.1016/j.str.2011.11.006">Phosphorylation at the interface.</a><br>
<b>Davis FP</b>. <i>Structure </i> (2011). 19(12):1726-1727 (commentary). (<a href="papers/Davis2011Structure.pdf">reprint</a>).</li><br>

<span class="anchor" id="Girard_etal11"></span>
<li value="16">
<a href="http://dx.doi.org/10.1111/j.1460-9568.2011.07918.x">Gene expression analysis in the parvalbumin-immunoreactive PV1 nucleus of the mouse lateral hypothalamus.</a><br>
Girard F, Meszar Z, Marti C, <b>Davis FP</b>, Celio M. <i>Eur. J. Neurosci</i> (2011). 34(12): 1934-1943. (<a href="papers/Girard2011EurJNeurosci.pdf">reprint</a>).</li><br>

<li value="15">
<a href="http://dx.doi.org/10.1039/C0MB00200C">Proteome-wide prediction of overlapping small molecule and protein binding sites using structure.</a><br>
<b>Davis FP</b>. <i>Molecular Biosystems</i> (2011). 7(2): 545-557. (<a href="papers/Davis2011MolBiosyst.pdf">reprint</a>).<br>
Selected for the <a href="http://scitation.aip.org/getabs/servlet/GetabsServlet?prog=normal&id=VIRT02000021000003000041000001&idtype=cvips&gifs=yes&ref=no"><i>Virtual Journal of Biological Physics Research</i></a>.<br>
Software: <a href="http://fredpdavis.com/homolobind">HOMOLOBIND</a></li><br>

<li value="14">
<a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1000668">The overlap of small molecule and protein binding sites within families of protein structures.</a><br>
<b>Davis FP</b> and Sali A. <i>PLoS Computational Biology</i> (2010) 6(2): e1000668. (<a href ="papers/Davis2010PLoSCompBiol.pdf">reprint</a>).<br> Designated feature research.<br>
Database: <a href="http://fredpdavis.com/pibase.ligands">PIBASE.ligands</a></li><br>

<span class="anchor" id="Davis_Eddy09"></span>
<li value="13">
<a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btp288">A tool for identification of genes expressed in patterns of interest using the Allen Brain Atlas.</a><br>
<b>Davis FP</b> and Eddy SR. <i>Bioinformatics</i> (2009) 25(13):1647-54. (<a href="papers/Davis2009Bioinf.pdf">reprint</a>).<br>
Software: <a href="http://fredpdavis.com/allenminer">ALLENMINER</a></li><br>

<li value="12">
<a href="http://dx.doi.org/10.1093/nar/gkn791">MODBASE, a database of annotated comparative protein structure models and associated resources</a><br>
Pieper U, Eswar N, Webb BM, Eramian D, Kelly L, Barkan DT, Carter H, Mankoo P, Karchin R, Marti-Renom MA, <b>Davis FP</b>, Sali A. <i>Nucleic Acids Res</i> (2008). (<a href="papers/Pieper2008NAR.pdf">reprint</a>).<br>
Database: <a href="http://salilab.org/modbase">MODBASE</a></li><br>

<li value="11">
<a href="http://www.proteinscience.org/cgi/doi/10.1110/ps.073228407">Host-pathogen protein interactions predicted by comparative modeling.</a><br>
<b>Davis FP</b>, Barkan DT, Eswar N, McKerrow JH, Sali A. <i>Protein Science</i> (2007) 16: 2585-2596. (<a href="papers/Davis2007ProteinSci.pdf">reprint</a>) (<a href="papers/Davis2007ProteinSci_suppl.pdf">supplementary text</a>).<br>
Software: <a href="http://fredpdavis.com/modtie">MODTIE</a>.<br>
Data: <a href="http://dx.doi.org/10.5281/zenodo.29608"><img src="https://zenodo.org/badge/doi/10.5281/zenodo.29608.svg"/></a></li><br>

<li value="10">
<a href ="http://dx.doi.org/10.1093/nar/gkm236">DBAli tools: mining the protein structure space.</a><br>
Marti-Renom MA, Pieper U, Madhusudhan MS, Rossi A, Eswar N, <b>Davis FP</b>, Al-Shahrour F, Dopazo J, and Sali A. <i>Nucleic Acids Research</i> (2007) 35: W393-7. (<a href="papers/MartiRenom2007NAR.pdf">reprint</a>).</li><br>

<li value="9">
<a href ="http://www.biomedcentral.com/1471-2105/8/S4/S4">The AnnoLite and AnnoLyze Programs for Comparative Annotation of Protein Structures.</a><br>
Marti-Renom MA, Rossi A, Al-Shahrour F, <b>Davis FP</b>, Pieper U, Dopazo J, and Sali A. <i>BMC Bioinformatics</i> (2007) 8:S4. (<a href="papers/MartiRenom2007BMCBioinf.pdf">reprint</a>)</li><br>

<li value="8">
<a href ="http://dx.doi.org/10.1371/journal.pcbi.0020153">Modeling protein interactions by analogy: Application to PSD-95.</a><br>
Korkin D, <b>Davis FP</b>, Alber A, Luong T, Shen MY, Lucic V, Kennedy MB, Sali A. <i>PLoS Computational Biology</i> (2006) 2: e153. (<a href="papers/Korkin2006PLoSCompBiol.pdf">reprint</a>)</li><br>

<li value="7">
<a href ="http://nar.oxfordjournals.org/cgi/content/full/34/10/2943">Protein complex compositions predicted by structural similarity.</a><br>
<b>Davis FP</b>, Braberg H, Shen MY, Pieper U, Madhusudhan MS, Sali A. <i>Nucleic Acids Research</i> (2006) 34:2943-2952. (<a href="papers/Davis2006NAR.pdf">reprint</a>).<br>
Software: <a href="http://fredpdavis.com/modtie">MODTIE</a></li><br>


<li value="6">
<a href ="http://dx.doi.org/10.1093/nar/gkj059">MODBASE: a database of annotated comparative protein structure models and associated resources.</a><br>
Pieper U, Eswar N, <b>Davis FP</b>, Braberg H, Madhusudhan MS, Rossi A, Marti-Renom MA, Karchin R, Webb BM, Eramian D, Shen MY, Kelly L, Melo F, Sali A. <i>Nucleic Acids Research</i> (2006) 34: D291-D295. (<a href="papes/Pieper2006NAR.pdf">reprint</a>).<br>
Database: <a href="http://salilab.org/modbase">MODBASE</a></li><br>


<li value="5">
<a href ="http://dx.doi.org/10.1110/ps.051571905">Localization of protein-binding sites within families of proteins.</a><br>
Korkin D, <b>Davis FP</b>, Sali A.  <i>Protein Science</i> (2005) 14: 2350-2360. (<a href="papers/Korkin2005ProteinSci.pdf">reprint</a>) </li><br>

<li value="4">
<a href ="http://dx.doi.org/10.1016/j.cplett.2005.02.029"> The Optimal Size of a Globular Protein Domain: A Simple Sphere-Packing Model.</a><br>
Shen MY, <b>Davis FP</b>, Sali A. <i>Chemical Physics Letters</i> (2005) 405: 224-228. (<a href="papers/Shen2005ChemPhysLett.pdf">reprint</a>) </li><br>

<li value="3">
<a href="http://dx.doi.org/10.1093/bioinformatics/bti277">PIBASE: a comprehensive database of structurally defined protein interfaces.</a><br>
<b>Davis FP</b> and Sali A. <i>Bioinformatics</i> (2005) 21(9): 1901-1907. (<a href="papers/Davis2005Bioinf.pdf">reprint</a>).<br>
Database: <a href="http://fredpdavis.com/pibase">PIBASE</a>.</li><br>

<li value="2">
<a href="http://dx.doi.org/10.1016/j.sbi.2004.04.006">A structural perspective on protein-protein interactions.</a><br>
Russell RB, Alber F, Aloy P, <b>Davis FP</b>, Korkin D, Pichaud M, Topf M, Sali A. <i>Current Opinions in Structural Biology</i> (2004) 14(3): 313-24. Review. (<a href="papers/Russell2004COSB.pdf">reprint</a>) </li><br>

<li value="1">
<a href="http://dx.doi.org/10.1093/nar/gkh095">MODBASE, a database of annotated comparative protein structure models, and associated resources.</a><br>
Pieper U, Eswar N, Braberg H, Madhusudhan MS, <b>Davis FP</b>, Stuart AC, Mirkovic N, Rossi A, Marti-Renom MA, Fiser A, Webb B, Greenblatt D, Huang CC, Ferrin TE, Sali A. <i>Nucleic Acids Res</i> (2004) 32 Database issue: D217-22. (<a href="papers/Pieper2004NAR.pdf">reprint</a>).<br>
Database: <a href="http://salilab.org/modbase">MODBASE</a></li><br>
</ol>

</td>
</tr>
</table>

<h3 id="software">Code</h3>

<table align="left"><tr><td align="left">
I've written several programs, mostly for protein structure and gene expression analysis, and all are <a href="http://github.com/fredpdavis?tab=repositories">available at github</a>.
I also publish the code I use to make figures in papers, <a href="#papers">linked above</a>.
<p>
</td></tr></table>

<table align="center" width=100%>

<tr>
<td align="right"><a href="http://fredpdavis.com/mdlabbook"><img width=60 style="border-width:0px" src="https://avatars2.githubusercontent.com/u/13946400?v=3&s=460"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/mdlabbook">mdllabbook</a> helps maintain a markdown-format lab notebook.</br>
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://fredpdavis.com/allenminer"><img width=60 style="border-width:0px" src="images/allenminer_web_logo.png"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/allenminer">ALLENMINER</a> - A tool to identify genes expressed in patterns of interest using the Allen Brain Atlas.<br>
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://fredpdavis.com/pibase"><img width=60 style="border-width:0px" src="images/pibase_blue_web.png"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/pibase">PIBASE</a> - A comprehensive relational database of structurally defined protein interfaces.<br>
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://fredpdavis.com/pibase.ligands"><img style="border-width:0px" width=60 src="images/pilig_web_logo.png"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/pibase.ligands">PIBASE.ligands</a> - Overlapping small molecule and protein binding sites.<br>
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://fredpdavis.com/modtie"><img style="border-width:0px" width=60 src="images/modtie_web_logo.png"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/modtie">MODTIE</a> - Software to predict protein complex compositions using structural similarity.<br>
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://dx.doi.org/10.5281/zenodo.29608"><img style="border-width:0px" width=60 src="images/hp_logo_web.png"></a></td>
<td><DIV class=bodytext>
<a href="http://dx.doi.org/10.5281/zenodo.29608">Predicted host-pathogen interactions</a> - Host-pathogen protein interactions predicted by comparative modeling.
</DIV></td>
</tr>

<tr>
<td align="right"><a href="http://fredpdavis.com/homolobind"><img style="border-width:0px" width=60 src="images/homolobind_logo.png"></a></td>
<td><DIV class=bodytext>
<a href="http://fredpdavis.com/homolobind">HOMOLOBIND</a> - proteome-wide prediction of protein and ligand binding sites using structure.
</DIV></td>
</tr>

</table>

</table>
</div>

<div class="footer">
<h3 id="contact">Contact:</h3>
email: fredpdavis at gmail dot com <br>
phone: (+1) 619-929-5892

</div>
</div>

</body>
</html>
